### PR TITLE
completed Toast

### PR DESCRIPTION
**Pull request by Adarsh-Patel-Dev**
Toast component : 
Success toast
Info Toast
Error toast

### CODE REVIEW

File: toast/toast.html [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+                        <li>
+                            <b>Error</b>
+
+                            <p>
+                                Toast with error message.
+                            </p>
+                        </li>
+                    </ul>
+                </div>
+            </p>
```
Comment (kotesh-arya) [inline toast/toast.html]: Just clear off these un-necessary gaps, rest is fine

**Review by kotesh-arya — APPROVED** [review]
Follow that one suggestion and you are good to go for a merge 👍🏻

File: toast/toast.html (hunk 1 above)
Comment (Adarsh-Patel-Dev) [reply toast/toast.html]: Thanks I'll remove that extra space

File: toast/toast.css [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    border-left: 0.5rem solid var(--BLUE);
+}
+
+.toast--success{
+    border-left: 0.5rem solid var(--GREEN);
+}
+
+.toast--icon{
+    font-size: 2rem;
+    padding: 5px;
```
Comment (theVivek-kumar) [inline toast/toast.css]: use rem in place of px

**Review by theVivek-kumar — CHANGES_REQUESTED** [review]
other things looks good

File: toast/toast.css (hunk 2 above)
Comment (Adarsh-Patel-Dev) [reply toast/toast.css]: done

**Pull request merged**